source/index.html.md

```diff
@@ -169,7 +174,46 @@ Get a FileGroup by making a `GET` request to `https://public.shipamax-api.com/ap
           "exportReference": string,
           "scac": string,
           "isRated": boolean,
-          "isDraft": boolean
+          "isDraft": boolean,
+          "shipper": string,
+          "shipperCWCode": string,
+          "consignee": string,
+          "consigneeCWCode": string,          
+          "notify": string,
+          "notifyCWCode": string,
+          "secondNotify": string,
+          "secondNotifyCWCode": string,
+          "destinationAgent": string,
+          "carrier": string,
+          "carrierCWCode": string,          
+          "grossWeight": string,
+          "grossVolume": string,
+          "vessel": string,
+          "vesselIMO": string,
+          "voyageNumber": string,
+          "loadPort": string,
+          "loadPortUnlocode": string,
+          "dischargePort": string,
+          "dischargePortUnlocode": string,
+          "shippedOnBoardDate": date,
+          "paymentTerms": string,
+          "category": string,
+          "subcategory": string,
+          "goodsDescription": string,
+          "transportMode": string,
+          "containerMode": string,
+          "shipmentType": string,
+          "container": [
+            {
+              "containerNo": string, 
+              "numberPieces": integer,
+              "pieceType": string,
+              "weight": string,
+              "volume": string,
+              "containerType": string,
+              "seals": string
```

Comment:
Just checked here that seals is a String list, at least when parsed

`
                        {
                            "billOfLadingId": 18,
                            "containerNo": "mrku0993443",
                            "numberPieces": null,
                            "pieceType": null,
                            "weight": null,
                            "volume": null,
                            "containerType": "40GP",
                            "seals": [
                                "CN0710623"
                            ]
                        },
                        {
                            "billOfLadingId": 18,
                            "containerNo": "mieu9999779",
                            "numberPieces": 190,
                            "pieceType": "CTN",
                            "weight": null,
                            "volume": null,
                            "containerType": "40HC",
                            "seals": [
                                "CN0901520"
                            ]
                        }`